uefi-std-example/src/main.rs

```diff
@@ -0,0 +1,26 @@
+#![feature(uefi_std)]
```

Comment:
Might want to add something to the CI to check that this builds, otherwise it could easily break at some point without us noticing.